cms/templates/bulkupdate.html

```diff
@@ -0,0 +1,133 @@
+
+      <div class="wrapper wrapper-status is-hidden">
+        <h3 class="title">${_("Bulk Update Status")}</h3>
+
+        <ol class="status-progress list-progress">
+          <li class="item-progresspoint item-progresspoint-submit is-complete">
+            <span class="deco status-visual">
+              <i class="icon fa fa-cog"></i>
+              <i class="icon fa fa-warning"></i>
+            </span>
+            <div class="status-detail">
+              <h3 class="title">${_("Applying")}</h3>
+              <p class="copy">${_("Applying your new settings")}</p>
+            </div>
+          </li>
+
+          <li class="item-progresspoint item-progresspoint-validate is-not-started">
+            <span class="deco status-visual">
+              <i class="icon fa fa-cog"></i>
+              <i class="icon fa fa-warning"></i>
+            </span>
+            <div class="status-detail">
+              <h3 class="title">${_("Validating")}</h3>
+              <p class="copy">${_("Validating your new settings")}</p>
+            </div>
+          </li>
+
+          <li class="item-progresspoint item-progresspoint-update-advanced-settings is-not-started">
+            <span class="deco status-visual">
+              <i class="icon fa fa-cog"></i>
+              <i class="icon fa fa-warning"></i>
+            </span>
+            <div class="status-detail">
+              <h3 class="title">${_("Updating Advanced Settings")}</h3>
+              <p class="copy">${_("Setting as default for future problems in Advanced Settings")}</p>
+            </div>
+          </li>
+
+          <li class="item-progresspoint item-progresspoint-update-problems is-not-started">
+            <span class="deco status-visual">
+              <i class="icon fa fa-cog"></i>
+              <i class="icon fa fa-warning"></i>
+            </span>
+            <div class="status-detail">
+              <h3 class="title">${_("Updating and Republishing Problems")}</h3>
+              <p class="copy">
+                  ${_("Applying update to all problems. Republishing published problems. This process might take longer with larger courses. You will receive an email when your update has completed.")}
+              </p>
+            </div>
+          </li>
+
+        </ol>
+      </div>
+    </article>
+
+    <aside class="content-supplementary" role="complimentary">
+      <div class="bit">
+        <h3 class="title title-3">${_("What is bulk update?")}</h3>
+        <p>
+          ${_("The bulk update page provides a way to update the Max Attempts and Show Answer settings for all existing problems of the current course. This operation also sets the default settings for all future problems of the current course. You can alter this default by going to Advanced Settings.")}
+        </p>
+      </div>
+    </aside>
+  </section>
+</div>
+
+<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js"></script>
```

Comment:
The platform already ships with its own copy of jquery.
We should probably use that, because this one (1.3) is awfully old anyway.

Grep for `jquery.min.js` for examples.

Reply:
done